buildSrc/src/main/groovy/org/elasticsearch/gradle/test/RestTestPlugin.groovy

```diff
@@ -34,6 +34,7 @@ class RestTestPlugin implements Plugin<Project> {
         integTest.configure {
             classpath = project.sourceSets.test.runtimeClasspath
             testClassesDir project.sourceSets.test.output.classesDir
+            cluster.distribution = 'zip' // rest tests should run with the real zip
```

Comment:
> The Java tests run against integ-test-zip but the REST tests run against zip.

No, java (unit) tests dont run against any distribution. Plugin integration tests run against integ-test-zip so that they are isolated (must be written without other plugins, even modules, in mind). QA tests use the full zip since they are trying to test the full system.